### ISSUE TITLE
SVGLoader transforms in Edge browser

### ISSUE THREAD

**Issue by OmarJay1** [mrdoob/three.js#16282]
Edge seems to incorrectly apply transforms for the SVGLoader.  For instance this appears different on Edge as opposed to Chrome & Firefox on my computer:

https://threejs.org/examples/#webgl_loader_svg

As far as I can tell Edge sometimes inserts white spaces after commas.  For instance:

<g transform="translate(200,200)" --> becomes <g transform="translate(200, 200)" 

and that breaks parseNodeTransform( node )

```javascript
var transformsTexts = node.getAttribute( 'transform' ).split( ' ' );
```

I have found the following fixes the problem:

```javascript
var tf = node.getAttribute( 'transform' );
tf = tf.replace(/\)/g, ')\t');
var transformsTexts = tf.split( '\t' );
```
Edge shouldn't be inserting white spaces, but I think splitting on white spaces is incorrect regardless.

There also seems to be an issue with SVGLoader not clearing transforms after Layers have been closed, but there are content workarounds for that.

Thanks.

**Comment by yomboprime** [comment]
> I have found the following fixes the problem:
> 
> ```js
> var tf = node.getAttribute( 'transform' );
> tf = tf.replace(/\)/g, ')\t');
> var transformsTexts = tf.split( '\t' );
> ```

Could you make a PR?
 
> There also seems to be an issue with SVGLoader not clearing transforms after Layers have been closed, but there are content workarounds for that.

Could you share a svg file with that issue?

Thanks.

**Comment by OmarJay1** [comment]
OK, I'll do that. I mentioned the issue of not clearing transforms after Layers are closed, and the above code seems to have made that a bigger issue.  I'm going to look at that and do some more extensive testing before doing the PR.  Thanks.

**Comment by OmarJay1** [comment]
This is simpler and seems to work better.  I'll do some more testing and then the PR.  Thanks.

```javascript
var transformsTexts = node.getAttribute( 'transform' ).split( ') ' );
```

**Comment by Mugen87** [comment]
Fixed via #16322
